R/get-net-task.R

```diff
@@ -1,19 +1,28 @@
-#' Returns an instance of the specified .NET Task
+#' Get an Instance of the Specified `.NET` Task
 #'
-#' @param taskName The name of the task to retrieve (without the Get)
+#' @param taskName The name of the task to retrieve (**without** `Get` prefix).
 #'
-#' @return An instance of the Task
+#' @return
+#'
+#' - `getNetTask()` returns an instance of of the specified `.NET` task.
+#' - `getContainerTask()` returns a reference to the container task for
+#' optimization purposes only.
+#'
+#' @examples
+#'
+#' if (FALSE) {
```

Comment:
@IndrajeetPatil Why FALSE? This should run without any problem no?

Reply:
I see:

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000
simulationLoader <- ospsuite:::getNetTask("SimulationLoader")
#> Error in rClr::clrCallStatic("OSPSuite.R.Api", paste0("Get", taskName)): Type:    System.MissingMethodException
#> Message: Could not find a suitable static method GetSimulationLoader on type OSPSuite.R.Api without method parameters
#> Method:  Void ThrowMissingMethod(System.Type, System.String, System.String, System.Type[])
#> Stack trace:
#>    at Rclr.ReflectionHelper.ThrowMissingMethod(Type classType, String methodName, String modifier, Type[] types)
#>    at Rclr.ClrFacade.InternalCallStaticMethod(Type classType, String methodName, Boolean tryUseConverter, Object[] arguments)
#>    at Rclr.ClrFacade.CallStaticMethod(String typename, String methodName, Object[] arguments)
```

<sup>Created on 2022-04-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

Reply:
Let's use a class that exists then.
SimulationRunner for instance is one of them
so 
`simulationRunner <- getNetTask("SimulationRunner")`

And what does the ::: does again?

Reply:
`:::` is a way to access non-exported (internal-only, i.e.) package objects.

That works, thanks! I'll change the example.

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000
simulationLoader <- ospsuite:::getNetTask("SimulationRunner")
```

<sup>Created on 2022-04-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

Reply:
Added this new example, which is no longer skipped.